### DISCUSSION TITLE
Support for window 11 ?

### DISCUSSION THREAD

**Discussion by kiwnatthawut** [erfg12/memory.dll#151] [Q&A] [answered]
Memory.dll this function support for window 11 ?

**Comment by erfg12** [comment, marked as answer]
I have never had an issue with windows 11. It’s my daily driver machine.

**Reply by kiwnatthawut** [reply]
i have  some issue for window 11 
what program support for use memory.dll in window 11 ?
.netframework version ? 
or other program

**Comment by erfg12** [comment]
It’s .NET standard, which means it works for framework and core.

**Reply by kiwnatthawut** [reply]
Ya im use framwork and core version 4.0 and 3.0
but it not support memory.dll 
How to download .NET standard ?
i will download it for test

**Reply by erfg12** [reply]
Idk what you mean. It’s already .NET standard, nothing has changed.

**Reply by kiwnatthawut** [reply]
i mean i will download it but idk how to download it
how to download it ? 
.NET standard  2.0

**Reply by kiwnatthawut** [reply]
open process can't work for memory.dll in window 11
my issue

**Reply by erfg12** [reply]
Read our wiki to get started.

**Comment by KleanOni** [comment]
this is probably a dumb question but some users who are non-english users who try using my trainer but when writing floats for some reason its being written with comma's instead of decimals unless they change all their language settings to english to fix. not sure if its a memory.dll issue or my trainer settings.
